src/api/procedures/__tests__/utils.ts

```diff
@@ -609,7 +609,7 @@ describe('authorization request validations', () => {
   });
 
   describe('assertAuthorizationRequestValid', () => {
-    test('should throw with an expired request', () => {
+    it('should throw with an expired request', async () => {
```

Comment:
Is it necessary for these to be async?

Reply:
No, I stripped them out before, but got hit with a merge conflict that reintroduced them.